pkg/common/ac_operations.go

```diff
@@ -278,8 +279,17 @@ func (a *ACOperationsImpl) tryToFindAC(acs []*accrd.AvailableCapacity, storageCl
 	var (
 		allocatedCapacity int64 = math.MaxInt64
 		foundAC           *accrd.AvailableCapacity
+		driveUUID         = make([]string, 0)
 	)
 	for _, ac := range acs {
+		// Available capacity with system disk location won't be allocated
```

Comment:
When system drive has free space nothing prevents us of using it.

Reply:
Fixed